api/desecapi/models.py

```diff
@@ -339,7 +339,22 @@ def is_registrable(self):
     @property
     def keys(self):
         if not self._keys:
-            self._keys = pdns.get_keys(self)
+            self._keys = [{**key, 'managed': True} for key in pdns.get_keys(self)]
+            try:
+                unmanaged_keys = self.rrset_set.get(subname='', type='DNSKEY').records.all()
+            except RRset.DoesNotExist:
+                pass
+            else:
+                name = dns.name.from_text(self.name)
+                for rr in unmanaged_keys:
+                    key = dns.rdata.from_text(dns.rdataclass.IN, dns.rdatatype.DNSKEY, rr.content)
+                    self._keys.append({
+                        'dnskey': rr.content,
+                        'ds': [dns.dnssec.make_ds(name, key, algo).to_text() for algo in (2, 4) if key.flags & 0x0001],
```

Comment:
Use 

```python3
>>> import dns.rdtypes.ANY.DNSKEY
>>> dns.rdtypes.ANY.DNSKEY.SEP
<Flag.SEP: 1>
```

instead of `0x0001` to avoid magic number

Reply:
done